api/src/controllers/record-utils.js

```diff
@@ -4,7 +4,8 @@ const _ = require('underscore'),
       config = require('../config'),
       smsparser = require('../services/report/smsparser'),
       validate = require('../services/report/validate'),
-      PublicError = require('../public-error');
+      PublicError = require('../public-error'),
+      DATE_NUMBER_STRING = /\d{13,}/;
```

Comment:
Seems to reject dates at or before `2001-09-09T01:46:39.999Z`.

Reply:
This is only used for sent timestamps which should only be after 2001. It's also consistent with what was there before so presumably is working in production. I think it's probably ok but it feels pretty ugly.